### PR TITLE
Update building instructions for OSX

### PR DESCRIPTION
**Pull request by mangelajo**

### CODE REVIEW

File: src/content/development/building-testing/_index.en.md [hunk 1]
```diff
@@ -112,3 +112,30 @@ To jump into a shell in Submariner's standard development environment:
...
+
+```
+sed: 1: "/"DAPPER_RUN_ARGS=/{s/[ ...": extra characters at the end of q command
+docker: invalid reference format: repository name must be lowercase.
+```
+
+This problem is seen in OSX while running make because the OSX shipped `getopt` tool is very limited.
+To fix this issue you need to install gnu-getopt:
+
+`brew install gnu-getopt`
```
Comment (skitt) [inline src/content/development/building-testing/_index.en.md:135]: I’ve reworked https://github.com/submariner-io/shipyard/pull/594 to avoid `getopt` ;-).

Comment (mangelajo) [reply src/content/development/building-testing/_index.en.md:135]: Oh sorry i think I was misleading, it's something in sed. Getopt is also used by shflags and has the same issue, so feel free to keep getopt implementation if it's better, we will need it anyway

File: src/content/development/building-testing/_index.en.md [hunk 2]
```diff
@@ -112,3 +112,30 @@ To jump into a shell in Submariner's standard development environment:
 ```shell
 make shell
 ```
+
+## Known issues
+
+### OSX getopt and sed are incompatible with our scripts
+
+symptoms:
+```
```
Comment (dfarrell07) [inline src/content/development/building-testing/_index.en.md:121]:
```suggestion
symptoms:

```text
```

File: src/content/development/building-testing/_index.en.md [hunk 3]
```diff
@@ -112,3 +112,30 @@ To jump into a shell in Submariner's standard development environment:
...
+  flags:ERROR short flag required for (tag) on this platform and "docker buildx build" requires exactly 1 argument
+```
+
+or
+
+```
```
Comment (dfarrell07) [inline src/content/development/building-testing/_index.en.md:127]:
```suggestion
```text
```

**Review by dfarrell07 — APPROVED** [review]
Those suggestions are what's failing the MD linter, otherwise +1.

**Comment by dfarrell07** [comment]
Do we still want this?

**Pull request closed without merging**